packages/react-dom/test-utils.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+if (process.env.NODE_ENV !== 'production') {
+  module.exports = require('./cjs/test-utils.development.js');
```

Comment:
I considered throwing, but I thought it would be immediately obvious when you tried to use it. I guess I can throw a more meaningful error message at least, sure!